gradle.properties

```diff
@@ -16,7 +16,7 @@ kotlin.code.style=official
 android.useAndroidX=true
 android.enableJetifier=false
 android.enableUnitTestBinaryResources=false
-org.gradle.parallel=false
+org.gradle.parallel=true
```

Comment:
There's actually a commented out `org.gradle.parallel=true` on line 13. Can we remove the current and uncomment the above since it provides more documentation to what the property does?